### PR TITLE
Download operation fix, docker configurations made & dependency fix

### PR DESCRIPTION
**Pull request by liu-hanli**

### CODE REVIEW

**Review by franchu7 — COMMENTED** [review]
It works! Good job!

File: openelevationservice/server/db_import/filestreams.py [hunk 1]
```diff
@@ -31,43 +31,37 @@ def downloadsrtm(xy_range):
...
-                log.debug("Downloaded file {} to {}".format(link.text, TILES_DIR))
+    # pw = environ.get('SRTMPASS')
+    # user = environ.get('SRTMUSER')
+    # if not user and not pw:
+    #     auth = tuple(SETTINGS['srtm_parameters'].values())
+    # else:
+    #     auth = tuple([user,pw])
+    # session.auth = auth
+    
+    # log.debug("SRTM credentials: {}".format(session.auth))
```
Comment (carloshellin) [inline openelevationservice/server/db_import/filestreams.py:42]: Remove comments if not needed

**Review by carloshellin — APPROVED** [review]
Good!

File: openelevationservice/server/db_import/filestreams.py (hunk 1 above)
Comment (liu-hanli) [reply openelevationservice/server/db_import/filestreams.py:42]: DONE

**Pull request merged**